src/main/java/shop/sajotuna/order/orders/repository/OrderRepository.java

```diff
@@ -1,11 +1,19 @@
 package shop.sajotuna.order.orders.repository;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import shop.sajotuna.order.orders.entity.Order;
+import shop.sajotuna.order.orders.entity.OrderStatus;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 
 public interface OrderRepository extends JpaRepository<Order, Long> {
     List<Order> findByUserId(long userId);
+
+    List<Order> findOrdersByStatus(OrderStatus orderStatus);
+
+    @Query("SELECT o FROM Order o WHERE o.status = 'SHIPPED' AND o.shippingDate <= :time")
+    List<Order> findShippedOrders(LocalDateTime time);
```

Comment:
```suggestion
List<Order> findByStatusAndShippingDateLessThanEqual(OrderStatus status, LocalDateTime time);
```
직접 쿼리를 작성하기 보다는 이런 쿼리 메서드나 QueryDsl을 활용해 보는것도 괜찮아 보입니다